step.yml

```diff
@@ -0,0 +1,172 @@
+---
+title: Create App Store Release
+summary: The Step creates an App Store release with the Release Management feature.
+description: |-
+  Create and configure a new release to App Store Connect with Release Management.
+website: https://github.com/bitrise-steplib/bitrise-step-create-new-app-store-release
+source_code_url: https://github.com/bitrise-steplib/bitrise-step-create-new-app-store-release
+support_url: https://github.com/bitrise-steplib/bitrise-step-create-new-app-store-release/issues
+
+type_tags:
+- deploy
+
+project_type_tags:
+- ios
+- macos
+- react-native
+- cordova
+- ionic
+- flutter
+
+toolkit:
+  go:
+    package_name: github.com/bitrise-steplib/tmp-bitrise-step-create-new-app-store-release
+
+inputs:
+- bundle_id:
+  opts:
+    title: Bundle identifier
+    summary: The bundle ID of the app to be released.
+    description: |-
+      Release Management requires the bundle ID of the iOS application being released to the App Store Connect.
```

Comment:
The description should start with the summary and add more details to it if any.